lib/active_record/mysql/enum/validations.rb

```diff
@@ -13,18 +13,18 @@ def validates_columns(*column_names)
           column_names.each do |name|
             col = cols[name.to_s]
             raise ArgumentError, "Cannot find column #{name}" unless col
-            
+
             # test for nullability
             validates_presence_of(name) if !col.null
-            
+
             # Test various known types.
             case col.type
             when :enum
               validates_inclusion_of name, :in => col.limit, :allow_nil => true
```

Comment:
This looks like a bug that it passes `allow_nil: true`. Shouldn't it plumb the flag from `col.null`?
```
validates_inclusion_of name, :in => col.limit, :allow_nil => col.null
```

Reply:
It does look like a bug, but without a great way to run tests, I'd like to leave the code as is right now, and work to improve it once we've got unit tests up and running.  I've created a ticket already to add such unit tests and modernize the project a bit.

Reply:
Ok, I'm fine with the fix coming in then.